### DISCUSSION TITLE
Let's discuss: i18n-ize your app today!

### DISCUSSION THREAD

**Discussion by amruthpillai** [github/india#41] [Open Source]
Hey everyone! 👋🏻 

Thank you for tuning into my talk at GitHub Satellite on internationalization and the importance of including multiple languages and cultures in open-source apps. I hope you found it exciting and informative.

You can check out Reactive Resume's GitHub Repository here: https://github.com/AmruthPillai/Reactive-Resume
The link to build your own resume is right here: https://rxresu.me

Here is the link to my presentation shared through the talk: https://docs.google.com/presentation/d/1hkrkU7wWWFKVqBE1JHq0HjDwliEt_zmoVcgyq4qjoYE/edit?usp=sharing

And I'd love to answer any questions you might have on any topic right here on this thread :)
Thank you to everyone who tuned it! ❤️ 

--
**Amruth Pillai**
www.amruthpillai.com
hello@amruthpillai.com

**Comment by gautamkrishnar** [comment]
Cool project 😄

**Reply by amruthpillai** [reply]
Thank you so much @gautamkrishnar! :)

**Reply by Saicharan-1996** [reply]
Hi sir,
I saw your portfolio Website its so awesome and inspiring for the young developers

**Comment by Saicharan-1996** [comment]
I would like to know what is the mindset as the developer we should have to start a opensource project and how balance our activities/hobbies along with our projects, jobs etc. by see your portfolio is fully diversified

**Reply by amruthpillai** [reply]
In a nutshell, I can just say you should be as curious as you can be. My entire philosophy to learning everything has been that it seemed interesting, and I wanted to learn more about it. Doesn't matter if it is useful or will it help me land a job, all that should be non-existent. Just focus on learning new things for the joy of learning, and somewhere down the road, it will help you, be it for a job, or when some idea strikes you to build your own project.

**Reply by Saicharan-1996** [reply]
Thank you sir ,what you said is correct .your answer is inspiring to me.

**Comment by akash-pal** [comment]
Hi Amruth,
Impressive project. 
_Its a one stop solution for creating resumes._ 
I used to reply on some paid tools to create resume but now finally have an option. 
I will be sharing it with my colleagues this definitely. 

In the past i have tried to implement automating i18n.
One challenge i had came across was most of the translation services are paid.
From your talk I can see you have used **Crowdin**.

Is there any limitation in terms of you have come across while using **Crowdin**?

**Reply by amruthpillai** [reply]
One of the key reasons to include i18n into your project in the initial stages of product developments, if possible, is that you won't be stalled due to any limitations of the platform.

Crowdin has been extremely good to me, and not only that, but it also offers premium services for free for all open-source projects, you just need to reach out to them. I did try out others as well, but Crowdin seemed the most simplest for me to use, and therefore, should be simpler for users who are not tech-savvy, but are capable of translating from one language to another.

**Reply by akash-pal** [reply]
Sounds great with Crowdin. 

There are circumstances when the translation provided by translation services are not in the proper context.
Have you come across any such issue?

Have you come around to thinking of this feature [Linkedin Integration](https://github.com/AmruthPillai/Reactive-Resume/issues/32) ?

**Reply by amruthpillai** [reply]
This was going to be one of the main features when I released the v2 version of the app, but unfortunately, I did not find any reliable API that could collect LinkedIn information in an organized format to be able to use in the app. I tried using a lot of open source repos on GitHub that advertise to do the same, to scrape LinkedIn for profile information, but none of them worked as expected.

This is still one of the most requested features and will get around to building a reliable LinkedIn profile copy mechanism as soon as the APIs are available from LinkedIn themselves to do this, the legal route.

**Reply by akash-pal** [reply]
Will definitely keep a lookout for this feature.
Also adding more templates would give more options.

Best of luck Amruth. 👍
